packages/react/src/components/Radio/index.tsx

```diff
@@ -211,43 +99,46 @@ const RadioGroupContext = React.createContext<RadioGroupContext>({
 })
 
 export function RadioGroup<Value extends string = string>({
```

Comment:
This might be a good opportunity to switch to `forwardRef`. In addition, make sure to pass the rest props to the div.